large_job_submission.py

```diff
@@ -0,0 +1,77 @@
+"""Submits simulations to the API."""
+import os
+
+import numpy as np
+
+from lib import models
+from lib import scenarios
+
+import inductiva
+
+INPUT_DATASET = "Path/to/meshes"
```

Comment:
we don't have this folder.

Reply:
Should I add to this repo a folder with 5 or so metaballs?

Reply:
One metaball or we can use the motorbike object from OpenFOAM, which is more classical.